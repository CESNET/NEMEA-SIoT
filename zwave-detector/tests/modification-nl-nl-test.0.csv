time,time TIMESTAMP,uint8 CHANNEL,bytes FRAME
2020-01-19T22:57:00,2020-01-19T22:57:00.000,1,00
2020-01-19T22:57:02,2020-01-19T22:57:02.000,1,e14534730241020c0501184d
2020-01-19T22:57:02,2020-01-19T22:57:02.000,1,e14534730503020a0210
