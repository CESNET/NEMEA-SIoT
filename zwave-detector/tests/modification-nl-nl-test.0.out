2,2020-01-19T22:57:02.000,2,"Unknown nodes in NL: 5,."
